takeover.py

```diff
@@ -55,6 +55,7 @@ def PERCENT(x, y):
         "error": r"There isn\\\'t a Github Pages site here\.|a Github Pages site here"
     },
     "Shopify": {"error": r"Sorry\, this shop is currently unavailable\."},
+    "Shopify": {"error": r"Sorry\, this store is currently unavailable\."},
```

Comment:
There are two keys with the same name `Shopify`. Since these are regular expressions, we can use something like: 
```python
"Shopify": {"error": r"Sorry\, this (shop|store) is currently unavailable\."}
```

Reply:
Done.